types/sauronjs/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for sauronjs 0.1
+// Project: https://github.com/Fullscript/sauronjs
+// Definitions by: EricPMulligan <https://github.com/EricPMulligan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.1
+
+import * as util from './util';
+export { cache, Component, instance, next, Service } from './core';
+
+interface SauronDOM {
+    update(): void;
+}
+
+interface SauronEvents {
+    dom: SauronDOM;
+}
+
+declare const events: SauronEvents;
```

Comment:
Instead of doing `declare const events: SauronEvents`, I recommend the following for clarity:
```ts
export namespace events {
  export namespace dom {
    export function update(): void;
  }
}
```

Reply:
Writing it like this, I get `strict-export-declare-modifiers` tslint error.

Reply:
Entered it as:

```
export namespace events {
  namespace dom {
    function update(): void;
  }
}
```